### PR TITLE
New tooltip design

### PR DESCRIPTION
**Pull request by witoszekdev**
I want to merge this because it updates the tooltip component to the new design. It also adds `storysource` and control `addons` to Storybook to make it easier to discover components and their usage from design library

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6743 Tooltip](https://app.clickup.com/t/2549495/SALEOR-6743)

File: src/Tooltip/Arrow.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { Side } from "@floating-ui/react-dom-interactions";
+import clsx from "clsx";
+import React, { forwardRef } from "react";
+
+import { makeStyles, useTheme } from "../theme";
+import { getBackgroundColor, getBorderColor, StyleProps } from "./styles";
+import { TooltipProps } from "./Tooltip";
+
+const useStyles = makeStyles<StyleProps>((theme) => ({
```
Comment (dominik-zeglen) [inline src/Tooltip/Arrow.tsx:9]: I think it can be moved to `styles.ts` as `useArrowStyles`

**Review by dominik-zeglen — APPROVED** [review]

File: src/Tooltip/Arrow.tsx (hunk 1 above)
Comment (witoszekdev) [reply src/Tooltip/Arrow.tsx]: 👍🏻 fixed that

**Review by Cloud11PL — APPROVED** [review]

**Comment by witoszekdev** [comment]
Resolved comments + fixed issues with hovering on border causing visual artifacts

**Pull request merged**